ts/packages/cli/src/helpers/synchronizers/logic/AzureFunctions.ts

```diff
@@ -0,0 +1,45 @@
+import { ClientSecretCredential } from '@azure/identity';
+import { WebSiteManagementClient } from '@azure/arm-appservice';
+import { EvaluatedConfigs } from '@configu/ts';
+import { CONFIG_SYNCHRONIZER_LABEL } from '@configu/lib';
+
+const label = CONFIG_SYNCHRONIZER_LABEL.AzureFunctions;
+
+export type AzureFunctionsConfiguration = {
+  // * Azure functions docs: https://learn.microsoft.com/en-us/azure/app-service/configure-common
+  tenantId: string;
+  clientId: string;
+  clientSecret: string;
+  subscriptionId: string;
+  resourceGroupName: string;
+  functionApp: string;
+};
+
+// TODO - Decide between syncAzureFunctionsConfigs (previous name) and syncAzureFunctionsAppSettings
+export const syncAzureFunctionsAppSettings = async ({
```

Comment:
@rannn505 We previously named this `syncAzureFunctionsConfigs`, Azure Functions names their 'configs' as 'application settings' so I thought `syncAzureFunctionsAppSettings` would be more appropriate here. Opinion?

Reply:
We agreed on `syncAzureFunctionsAppSettings`.